src/interactions/zoom.js

```diff
@@ -100,6 +102,18 @@ extend(ChartInternal.prototype, {
 		// 	.call(z)
 		// 	.on("dblclick.zoom", null);
 
+		if ($$.zoomScale) {
+			const range1 = $$.zoomScale.domain()[0];
+			const range2 = $$.x.domain()[0];
+			const delta = 0.015;
```

Comment:
what is delta?

Reply:
when zoom is out, the domain value can have a decimal differences due to user's wheel action.
ex. the initial domain is 0.05, but zomming out domain value is like 0.049.....5.

The delta value is to determine if user reached at the start domain value.

Reply:
@netil Ok I got it.